### PR TITLE
Tidy non-telecomms radio code

### PR DESCRIPTION
**Pull request by SpaceManiac**
:cl:
code: Remote signaler and other non-telecomms radio code has been cleaned up.
/:cl:

Follow-up to #33236. Cleans up dead and badly named vars and procs, simplifies signal construction, and repaths the integrated signalers used by PDAs and pAIs. *Doesn't* (or shouldn't) change in-game functionality in any noticeable way.

### CODE REVIEW

File: code/controllers/subsystem/radio.dm [hunk 1]
```diff
@@ -14,35 +14,22 @@ SUBSYSTEM_DEF(radio)
...
 	var/datum/radio_frequency/frequency = frequencies[f_text]
-
 	if(frequency)
 		frequency.remove_listener(device)
-
-		if(frequency.devices.len == 0)
-			qdel(frequency)
-			frequencies -= f_text
-
+		// let's don't delete frequencies in case a non-listener keeps a reference
```
Comment (vuonojenmustaturska) [inline code/controllers/subsystem/radio.dm]: Is this a thing that actually happens?

Comment (SpaceManiac) [reply code/controllers/subsystem/radio.dm]: I have no idea if deleting frequencies has ever caused a problem, but yes. The magnet controller and PDA signalers, for example. Of course, they could be changed.

**Comment by optimumtact** [comment]
fix the conflict please and I'll merge this

**Comment by SpaceManiac** [comment]
Conflict resolved.

**Pull request merged**